tooling/bundler/src/bundle/settings.rs

```diff
@@ -386,7 +384,6 @@ pub struct Settings {
 #[derive(Default)]
 pub struct SettingsBuilder {
   project_out_directory: Option<PathBuf>,
-  verbose: bool,
```

Comment:
we should mark this change as a breaking change in the change file, and show the solution (i know there's some people using the tauri bundler as a library)

Reply:
True! Good catch